Resources/Locale/en-US/_ES/station-events/gas-leak.ftl

```diff
@@ -0,0 +1,2 @@
+es-voter-query-string-gas-leak-gas = The gas that will be leaked is:
+es-voter-query-string-gas-leak-location = The gas will be leaked:
```

Comment:
```suggestion
es-voter-query-string-gas-leak-gas = The gas that should be leaked is:
es-voter-query-string-gas-leak-location = The gas should be leaked:
```

"will" kind of makes it seem like youre trying to guess where it'll go rather than influencing where it goes